lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -0,0 +1,48 @@
+import classnames from 'classnames';
+import { useParams } from 'wouter-preact';
+
+import type { StudentStats } from '../../api-types';
+import { useConfig } from '../../config';
+import { apiCall } from '../../utils/api';
+import { useFetch } from '../../utils/fetch';
+import StudentsActivityTable from './StudentsActivityTable';
+
+export default function DashboardApp() {
+  const { assignmentId = 'unknown' } = useParams();
+  const { dashboard, api } = useConfig(['dashboard', 'api']);
+  const assignment = {
+    title: dashboard.assignment.title,
+    id: assignmentId,
+  };
+  const students = useFetch<StudentStats[]>(`${assignmentId}_students`, () =>
+    apiCall({
+      authToken: api.authToken,
+      method: 'GET',
+      path: `/api/assignment/${assignmentId}/stats`,
```

Comment:
This should use the `urlPath` template tag to escape path components.

Reply:
I actually just realized this endpoint is returned by the BE in the dashboard config. It should not be hardcoded here.